lib/livebook_web/live/session_live/shortcuts_component.ex

```diff
@@ -113,6 +113,12 @@ defmodule LivebookWeb.SessionLive.ShortcutsComponent do
         seq_mac: ["⌘", "k"],
         press_all: true,
         desc: "Toggle keyboard control in cell output"
+      },
+      %{
+        seq: ["ctrl", "alt", "-"],
+        seq_mac: ["⌃", "⌥", "-"],
+        press_all: true,
+        desc: "Go back"
```

Comment:
Should we know love this to the universal secrion below?

I also think the description should say “Go to previous cell”.

Reply:
Doesn't it confuse the user, thinking this will behave like `k` shortcut (Focus previous cell)? Because VSCode also uses `Go back` for this feature.

Reply:
What about “Go back (cursor)”?

Reply:
What about `Go back to last editor`?

Reply:
I like "Go back to last editor"!